File: software/backend/src/main/java/ru/scms/back/dto/CertificateDto.java
package ru.scms.back.dto;

import java.time.LocalDateTime;
import ru.scms.back.enums.CertificateStatus;

public record CertificateDto(
        Long id,
        Long caseId,
        String caseCode,
        Long methUserId,
        String methUserName,
        String code,
        CertificateStatus status,
        String verificationCode,
        LocalDateTime issuedAt) {}

File: software/client-portal/src/shared/api/dto/certificateDto.ts
import type { CertificateStatus } from "./enums";

export interface CertificateDto {
  id: number;
  caseId: number;
  caseCode: string | null;
  methUserId: number;
  methUserName: string | null;
  code: string;
  status: CertificateStatus;
  verificationCode: string;
  issuedAt: string;
}
